ecms_base/features/custom/ecms_hotel/config/install/field.field.node.hotel.field_hotel_rate.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_hotel_rate
+    - node.type.hotel
+id: node.hotel.field_hotel_rate
+field_name: field_hotel_rate
+entity_type: node
+bundle: hotel
+label: 'Hotel Rate'
+description: ''
+required: true
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: string
```

Comment:
Just confirming this should be a `string` and not a `float`.

Reply:
Yes, the usage now is that it could show a price, or a percentage - and so string allows for that flexibility